ReSharper.FSharp/src/FSharp.TypeProviders.Host/src/TypeProvidersEndPoint.cs

```diff
@@ -32,6 +34,14 @@ protected override RdSimpleDispatcher InitDispatcher(Lifetime lifetime, ILogger
     protected override void InitLogger(Lifetime lifetime, string path)
     {
       ProtocolEndPointUtil.InitLogger(path, lifetime, LoggingLevel.TRACE);
+
+      if (Environment.GetEnvironmentVariable("INTERNAL_MODE") is { } env &&
```

Comment:
Does this process get all parent process environment variables? If yes, this logic could probably be a bit more defensive and use a prefix like `RESHARPER`, as "internal mode" is a bit too generic and could be set on computer.